### ISSUE TITLE
Reproduction of results

### ISSUE THREAD

**Issue by StrivedTye** [slothfulxtx/MBPTrack3D#13]
Hi,

Thanks for your efforts in this publication.

It seems that the results are heavily affected by the version of `pytorch `and `cuda`. For instance, when using `torch 1.12.1` and `cuda 11.7`, we can only obtain 91.52/66.26 in Pedestrian, and cannot reproduce the result of  93.9/68.6 reported in the paper.

Thanks.

**Comment by slothfulxtx** [comment]
Yes, other followers find the same problem. I've got no idea what happened. Based on my memory, I suspect this issue is caused by either the version of pytorch3d or CUDA version.

**Comment by ujaejoon** [comment]
@StrivedTye Have you been able to resolve the issue?

**Comment by StrivedTye** [comment]
> @StrivedTye Have you been able to resolve the issue?

No, I haven't. :(
